Project Walking/Source code/app/src/main/java/com/teamapricot/projectwalking/LocationHandler.java

```diff
@@ -0,0 +1,153 @@
+package com.teamapricot.projectwalking;
+
+import android.Manifest;
+import android.location.LocationListener;
+import android.location.LocationManager;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A class for accessing location data.
+ */
```

Comment:
Add @author and @version so that it's easy to see who created this class and when it was last changed, so that another team member do not accidentaly change this class without your permission.